autoload/db_ui/dbout.vim

```diff
@@ -127,6 +128,33 @@ function! db_ui#dbout#yank_header() abort
 endfunction
 
 function! s:get_cell_range(cell_line_number, curpos, scheme_name) abort
+  if a:scheme_name ==# 'oracle'
```

Comment:
This is where we detect whether or not we should use virtual columns in our function.